dnsaas/api/v2/tests.py

```diff
@@ -327,6 +328,58 @@ def test_post_raise_error_when_AAAA_content_is_invalid(self):
     #
     # updates
     #
+    def test_update_raise_error_when_record_has_no_owner_and_no_service(
```

Comment:
it's already here https://github.com/allegro/django-powerdns-dnssec/pull/208/files#diff-68a5eb82a9a2d178357d8415b055bb99R682
but test name perhaps could be better ;)
i'll fix the name